python/Ganga/Core/GangaThread/WorkerThreads/__init__.py

```diff
@@ -24,11 +24,14 @@ def shutDownQueues():
     logger = getLogger()
     logger.debug("Shutting Down Queues system")
     global _global_queues
-    _global_queues.lock()
-    _global_queues._purge_all()
+    try:
+        _global_queues.lock()
+        _global_queues._purge_all()
+    except:
+        logger.warning("Error in shutting down queues thread. Likely harmless")
     del _global_queues
     _global_queues = None
     import Ganga.GPI
+    setattr(Ganga.GPI, 'queues', None)
```

Comment:
You can't double delete things in Python. You can't even single-delete them. All `delattr` does is remove the reference to it.